src/xrCore/Threading/ThreadUtil.cpp

```diff
@@ -24,9 +24,15 @@ void SetThreadName(ThreadHandle threadHandle, pcstr name)
         DWORD dwFlags;
     };
 
+    auto namePrefix = "X-Ray ";
+    constexpr auto namePrefixSize = sizeof(namePrefix);
+    auto fullNameSize = sizeof(name) + namePrefixSize;
```

Comment:
sizeof is wrong here
```suggestion
    auto fullNameSize = xr_strlen(name) + namePrefixSize;
```